pkgs/by-name/wa/wayfreeze/package.nix

```
@@ -0,0 +1,33 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  libxkbcommon
+}:
+
+rustPlatform.buildRustPackage rec {
```

```suggestion
rustPlatform.buildRustPackage {
```

Unused when you remove the `repo = pname;` as suggested below.